src/SFA.DAS.ApplyService.InternalApi/Controllers/ApplicationController.cs

```diff
@@ -49,6 +50,13 @@ public async Task<ActionResult<Domain.Entities.Apply>> GetApplication(Guid appli
             return await _mediator.Send(new GetApplicationRequest(applicationId));
         }
 
+
+        [HttpGet("Application/FinancialReviewDetails/{applicationId}")]
```

Comment:
Routing should be: [HttpGet("Application/{applicationId}/FinancialReviewDetails")]